multiplayer/src/components/SignedOutPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useContext } from "react";
+import { showModal } from "../state/actions";
+import { AppStateContext } from "../state/AppStateContext";
+import { useAuthDialogMessages } from "../hooks/useAuthDialogMessages";
+
+export default function Render() {
+    const { dispatch } = useContext(AppStateContext);
+
+    const dialogMessages = useAuthDialogMessages();
+
+    useEffect(() => {
+        dispatch(showModal("sign-in", { dialogMessages }));
+    }, [dialogMessages]);
+
+    return <></>;
```

Comment:
Is this any different from returning null? (Just curious, no need to update or change anything).

Reply:
It's effectively the same as returning null. Fragments themselves don't add anything to the DOM, so it's a no-op.